ush/HWP_tools.py

```diff
@@ -7,6 +12,14 @@
 import fnmatch
 
 def check_restart_files(hourly_hwpdir, fcst_dates):
+    """Check the restart files.
+
+    Args:
+         hourly_hwpdir: the directory where the RESTART data is copied (nwges/HOURLY_HWP)
+         fcst_dates: Forecast dates
+    Returns:
+         list of hwp_avail_hours, hwp_non_avail_hours

```

Comment:
@jordanschnell  Would the returned value be lists of hours with and without available RESTART files?

Reply:
Correct